spec/helpers/budget_warning_spec.rb

```diff
@@ -24,12 +24,10 @@
 
   describe '#whitelisted pages' do
     context 'english page for tools' do
-      ['redundancy-pay-calculator',
-        'workplace-pension-contribution-calculator',
-        'pension-calculator'].each do |tool_name|
-            it_behaves_like 'displays_banner_warning_for_whitelisted_tools' do
-               let(:url) { "www.example.com/en/tools/#{tool_name}/" }
-            end
+      ['redundancy-pay-calculator'].each do |tool_name|
```

Comment:
Why loop here? Are you expecting to have re-add more calculators? Even if so it'd probably be worth it to remove the looping until then.